XML/TestCases/FunctionalTests-TVM.xml

```diff
@@ -14,4 +14,23 @@
       <VMGeneration>2</VMGeneration>
     </SetupConfig>
   </test>
+  <test>
+    <testName>TVM-TEST-COMPATIBILITY</testName>
+    <testScript>TVM-TestCompatibility.sh</testScript>
+    <files>.\Testscripts\Linux\TVM-TestCompatibility.sh,.\Testscripts\Linux\utils.sh</files>
+    <Platform>Azure</Platform>
+    <Category>Functional</Category>
+    <Area>TVM</Area>
+    <Tags>tvm</Tags>
+    <Priority>1</Priority>
+    <SetupConfig>
+      <TestLocation>eastus2euap</TestLocation>
+      <SetupType>OneVM</SetupType>
+      <SecureBoot>true</SecureBoot>
+      <vTPM>true</vTPM>
+      <OverrideVMSize>Standard_D2s_v3,=~Standard_D[0-9]+s_v3</OverrideVMSize>
+      <VMGeneration>2</VMGeneration>
+      <ARMImageName>=~0003-com-ubuntu-server-trusted-vm\s18_04-gen2</ARMImageName>
```

Comment:
seems the test case content is trying to handle more compatible distro images, like redhat, centos, oracle, debian, ubuntu, suse, etc.  So, this kind of setting for ARMImage constraint doesn't make any sense.

Reply:
Currently there is only one distro and image that supports this test case. None of the other images in gallery support this test. We can add more images as they onboard to TVM. Also, the primary use case of this test is with OS VHDs of the distro images that are requesting to onboard to TVM.